### PR TITLE
Adds information on the "Update Many Repos" process

### PR DESCRIPTION
**Pull request by Gowiem**
## What

1. Adds a new doc to `community/` called contributor-how-tos which I'm proposing to be our repository of guides for complicated contributor administrivia.
    - Note: I'm totally open to put this elsewhere. Just wasn't sure where it would fit!
1. Adds docs on the "Update Many Repos" process.
1. Fixes nuru's avatar as I noticed it was incorrectly copy pasta'd.

## Why

1. So other contributors know how to use microplane and I'm not a silo of doing the "Update Many Repos" process.

### CODE REVIEW

File: content/community/contributor-how-tos.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Contributor How-Tos
```
Comment (osterman) [inline content/community/contributor-how-tos.md:2]:
```suggestion
title: Contributor Tips & Tricks
```

File: content/community/contributor-how-tos.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+description: ''
+---
+
+## About
+
+This document is intended to describe common and not-so-common processes that the contributor team executes as part of maintaining the 300+ open source projects within the Cloud Posse Organization.
+
+
+## How-Tos
```
Comment (osterman) [inline content/community/contributor-how-tos.md:11]:
```suggestion
## Tips & Tricks
```

File: content/community/contributor-how-tos.md (hunk 1 above)
Comment (osterman) [reply content/community/contributor-how-tos.md:2]: can you rename file to `contributor-tips.md`

File: content/community/contributor-how-tos.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+### Update Many Repos
```
Comment (osterman) [inline content/community/contributor-how-tos.md:13]:
```suggestion
### Update Multiple Repos at Once
```

File: content/community/contributor-how-tos.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+To update many of the open source repos with a common change such as updating Terraform `required_version` pinning, adding GitHub actions, or updating pinned providers, the contributor team has adopted using [microplane](https://github.com/Clever/microplane). This tool allows us to execute automated changes across dozens or even hundreds of our open source repos, which saves many man hours of contributor time.
```
Comment (nitrocode) [inline content/community/contributor-how-tos.md:15]:
```suggestion
To update many of the open source repos with a common change such as updating Terraform `required_version` pinning, adding GitHub actions, or updating pinned providers, the contributor team has adopted using [microplane](https://github.com/Clever/microplane). This tool allows us to execute automated changes across dozens or even hundreds of our open source repos, which saves many hours of contributor time.
```

nitpick: More inclusive by omitting the word `man`

File: content/community/contributor-how-tos.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    >
+    > 1. $INFO_ON_YOUR_CHANGE_NUMBER_ONE
+    > 1. $INFO_ON_YOUR_CHANGE_NUMBER_TWO
+    >
+    > ## Why
+    >
+    > 1. $INFO_ON_WHY_YOU_MADE_YOUR_CHANGE_NUMBER_ONE
+    > 1. $INFO_ON_WHY_YOU_MADE_YOUR_CHANGE_NUMBER_TWO" \
+    > -- \
+    > sh -c $PWD/$YOUR_SCRIPT_NAME.sh
```
Comment (osterman) [inline content/community/contributor-how-tos.md:51]:
~~~suggestion
1. Once you've got a script that is working like you expect, you can run the microplane plan command. This step executes the given script across all the repos specified in `init.json` and then commits the result. You should format your `mp plan` command as follows: 
    ```bash
    mp plan -b $YOUR_BRANCH_NAME -m "[AUTOMATED] $YOUR_PR_TITLE
    
    ## What
    
    1. $INFO_ON_YOUR_CHANGE_NUMBER_ONE
    1. $INFO_ON_YOUR_CHANGE_NUMBER_TWO
    
    ## Why
    
    1. $INFO_ON_WHY_YOU_MADE_YOUR_CHANGE_NUMBER_ONE
    1. $INFO_ON_WHY_YOU_MADE_YOUR_CHANGE_NUMBER_TWO" \
    -- \
    sh -c $PWD/$YOUR_SCRIPT_NAME.sh
    ```
    __NOTE__ the quotes around the message passing this as an argument to `-m` (message)
~~~

File: content/community/contributor-how-tos.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+1. Add a [GH Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) to your shell's environment variables under the variable `GITHUB_API_TOKEN`:
+   1. `export GITHUB_API_TOKEN=$YOUR_TOKEN`
+1. Change to an empty directory that you can use as a scratch workspace for your Many Repos change:
+   1. `mkdir ~/mp_workspace && cd ~/mp_workspace`
+1. Initialize microplane:
+   1. `mp init --all-repos "cloudposse"`
+   1. Initializing creates an `mp/` folder in your current directoy, finds all of the Cloud Posse public projects, and then creates an `mp/init.json` file describing them.
+   1. NOTE: microplane supposedly has the ability to search against an organization and narrow the returned repos that end up in `init.json`, but that functionality seems buggy and not working. We do our repo filtering manually in the next step.
+1. Manually edit `mp/init.json` to only include the repos which you want to make changes against.
+   1. Your editor's multi-select and edit capabilities are your friend here!
```
Comment (osterman) [inline content/community/contributor-how-tos.md:31]:
```suggestion
   1. Your editor's multi-select and edit capabilities are your friend here! (or maybe some [`jq`](https://stedolan.github.io/jq/) if that's your thing)
```

File: content/community/contributor-how-tos.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+   1. Duplicate the original `init.json` so you don't need to re-run `mp init` in the case that you want to start fresh.
+1. Run microplane's clone command to pull all of the repos specified in `init.json` down to your local machine for mass changes:
+   1. `mp clone`
+1. Create a bash script to facilitate the changes that you're attempting to make against the many repos you've specified.
+   1. Use the microplane `-r` or `--repo` flag to operate on a single repo for testing your script prior to making the changes across all repos.
+   1. Go through the full microplane process (complete the following steps) for this single repo test-run and get it signed off by the other Cloud Posse contributors to ensure everyone agrees with the change prior to making it.
```
Comment (osterman) [inline content/community/contributor-how-tos.md:37]:
```suggestion
   1. Go through the full microplane process (complete the following steps) for this single repo test-run and get it signed off by the other Cloud Posse `#contributors` to ensure everyone agrees with the change prior to making it.
```

File: content/community/contributor-how-tos.md (hunk 3 above)
Comment (Gowiem) [reply content/community/contributor-how-tos.md]: Done.

File: content/community/contributor-how-tos.md (hunk 4 above)
Comment (Gowiem) [reply content/community/contributor-how-tos.md]: Done.

File: content/community/contributor-how-tos.md (hunk 5 above)
Comment (Gowiem) [reply content/community/contributor-how-tos.md]: Done.

File: content/community/contributor-how-tos.md (hunk 7 above)
Comment (Gowiem) [reply content/community/contributor-how-tos.md]: Done.

File: content/community/contributor-how-tos.md (hunk 6 above)
Comment (Gowiem) [reply content/community/contributor-how-tos.md]: Done.

**Pull request merged**